pkg/utils/proxy/cloudproxy.go

```diff
@@ -111,6 +111,10 @@ func (cp *CloudProxy) deploymentSpec() (v1apps.DeploymentSpec, error) {
 		},
 		Template: v1.PodTemplateSpec{
 			ObjectMeta: metav1.ObjectMeta{
+				Annotations: map[string]string{
```

Comment:
should we do this as a condition to the instance monitoring option?

Reply:
good point